site/gcs-cors.json

```diff
@@ -0,0 +1,19 @@
+[
+    {
+        "maxAgeSeconds": 3600,
+        "method": [
```

Comment:
We talked about the possibility of using customized verbs yesterday. Can we use a wildcard here for code extensibility?

Reply:
This is GCS only.